MinecraftClient/Commands/AutoAttack.cs

```diff
@@ -8,39 +8,35 @@ namespace MinecraftClient.Commands
     class AutoAttack : Command
     {
         public override string CMDName { get { return "autoattack"; } }
-        public override string CMDDesc { get { return "autoattack <on|off>: Enable/disable auto attack mobs."; } }
+        public override string CMDDesc { get { return "autoattack <on|off>: Auto attack mobs around you."; } }
 
         public override string Run(McTcpClient handler, string command)
         {
             if (hasArg(command))
             {
-                string state = getArg(command);
-                if (state.ToLower() == "on")
+                string arg = getArg(command);
+                if (arg == "on")
                 {
-                    if (!handler.AutoAttack)
+                    if (handler.GetEntityHandlingEnabled())
                     {
-                        handler.AutoAttack = true;
-                        return "Auto attack turned on.";
+                        // TODO: check if the bot already loaded
+                        // I don't know how :C
+                        handler.BotLoad(new ChatBots.AutoAttack());
+                        return "Auto Attack now enabled.";
```

Comment:
That kind of interaction is indeed not implemented yet, existing bots have an associated INI File setting like this:
```ini
[AutoRelog]
enabled=true
```
Then a single instance of the bot is loaded on launch if the setting is enabled, and that's it. You could maybe remove both commands and stick back to a static setting like other bots for now, then add support for dynamically enabling/disabling the built-in bots at a later time.

A way for that could be a a `/chatbot` command which could show current status of bots and allow the user to load/unload builtin bots. Of course McTcpClient would need to expose additional methods for allowing that.